src/serverless-functions/send-to-phone.ts

```diff
@@ -0,0 +1,43 @@
+import "source-map-support/register";
+
+import { EFNYC_HOST, serverlessRollbarHandler } from "../utils/serverless-util";
+import { Twilio } from "twilio";
+
+export const handler = serverlessRollbarHandler(async (event) => {
+  const {
+    TWILIO_ACCOUNT_SID,
+    TWILIO_AUTH_TOKEN,
+    TWILIO_PHONE_NUMBER,
+  } = process.env;
+  if (!(TWILIO_ACCOUNT_SID && TWILIO_AUTH_TOKEN && TWILIO_PHONE_NUMBER)) {
+    throw new Error("Twilio credentials are not in environment!");
+  }
+
+  const userPhone =
+    event.queryStringParameters && event.queryStringParameters.userPhone;
+
+  const resultsPagePath =
+    event.queryStringParameters && event.queryStringParameters.userPath;
+
+  if (!userPhone) {
+    return {
+      statusCode: 400,
+      body: "No receiving phone number given!",
+    };
+  }
+
+  const client = new Twilio(TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN);
+
+  await client.messages.create({
+    body: `Eviction Free NYC! Follow this link for assistance in your eviction case: https://${EFNYC_HOST}${
+      resultsPagePath || ""
+    }`,
```

Comment:
since you're not doing any validation of `resultsPagePath`, this actually has a bit of a security hole in it: an attacker could set up a domain like `www.evicitonfreenyc.org.evil.com` and pass a `resultsPagePath` of `.evil.com/somepage` to send the user to a domain they control!  So you might want to do some basic validation of `resultsPagePath`, e.g. test that it starts with a `/` at least.

Reply:
Ahhhh wow didn't even think of that!

Reply:
Also this attacker must be a pretty big deal if they own the domain `evil.com`...